.golangci.yml

```diff
@@ -19,6 +19,11 @@ linters:
   - varnamelen
   - paralleltest
   - cyclop
+  - exhaustruct
```

Comment:
temp disable some newly added lint checks, feel free to resolve on these lint issues